backend/papersfeed/utils/recommendations/utils.py

```diff
@@ -22,17 +27,33 @@ def select_user_actions(_):
         'UserId', 'ItemId', 'Type', 'Count'
     )
 
-    for action in new_actions:
-        paper_id = action['ItemId']
+    new_papers = Paper.objects.filter(
+        creation_date__gt=(datetime.now() + timedelta(days=-1))
+    ).annotate(
+        ItemId=F('id')
```

Comment:
[Not important] Why did you use camel casing?

Reply:
Because it will be sent to ml server!

Reply:
I can't get a point, could you explain more? So, ItemId is the exact name needed for ml codes?

Reply:
If it is true, why do we need to covert `id` to `ItemId` on this backend?

Reply:
Yes. In ml server, variable names are handled like this, ItemId, UserId ....